### PR TITLE
[kots]: add db migration job to fluent bit logger

### PR DESCRIPTION
**Pull request by MrSimonEmms**
## Description

Add DB migration job to FluentBit ephemeral log collector

## How to test

Install via KOTS, create a support bundle and search for `migrations-xxx.migrations` in the fluent bit logs

## Release Notes

```release-note
[kots]: add db migration job to fluent bit logger
```

## Werft options:

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [ ] /werft with-preview
- [ ] /werft with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`

### CODE REVIEW

**Comment by roboquat** [comment]
@MrSimonEmms: Adding the "do-not-merge/release-note-label-needed" label because no release-note block was detected, please follow our [release note process](https://git.k8s.io/community/contributors/guide/release-notes.md) to remove it.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Review by geropl — APPROVED** [review]
Approving that no sensitive data can be exposed via this logger :heavy_check_mark:

File: install/kots/manifests/gitpod-log-collector.yaml [hunk 1]
```diff
@@ -74,6 +74,13 @@ spec:
               Path /gitpod/log-collector
               Format plain
 
+          [OUTPUT]
+              Name file
+              Match migrations*
```
Comment (ArthurSens) [inline install/kots/manifests/gitpod-log-collector.yaml:79]: What exactly are we matching here? The pod name?

Comment (MrSimonEmms) [reply install/kots/manifests/gitpod-log-collector.yaml:79]: Yes - the file is in the format `migrations-<id.<container-name>`, so will pick up both `db-waiter` and `migrations` for the container name

Comment (ArthurSens) [reply install/kots/manifests/gitpod-log-collector.yaml:79]: Awesome, thanks for the info!

**Review by ArthurSens — APPROVED** [review]

**Pull request merged**